fdbserver/Status.actor.cpp

```diff
@@ -3393,9 +3393,14 @@ ACTOR Future<StatusReply> clusterGetStatus(
 			state std::vector<JsonBuilderObject> workerStatuses = wait(getAll(futures2));
 			wait(success(primaryDCFO));
 
-			std::vector<NetworkAddress> addresses =
-			    wait(timeoutError(coordinators.ccr->getConnectionString().tryResolveHostnames(), 5.0));
-			coordinatorAddresses = std::move(addresses);
+			ErrorOr<std::vector<NetworkAddress>> addresses =
+			    wait(errorOr(timeoutError(coordinators.ccr->getConnectionString().tryResolveHostnames(), 5.0)));
+			if (addresses.present()) {
+				coordinatorAddresses = std::move(addresses.get());
+			} else {
+				messages.push_back(JsonString::makeMessage("fetch_coordinator_addresses",
+				                                           "Fetching coordinators addresses timed out"));
```

Comment:
This is slightly different than the change on 71.3, where it says "coordinator" rather than "coordinators":

```suggestion
				                                           "Fetching coordinator addresses timed out"));
```

I think it may also be the case that shortening this one more character breaks the formatting